profiler/build.gradle.kts

```diff
@@ -7,6 +7,8 @@ dependencies {
   compileOnly("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api")
   compileOnly("io.opentelemetry:opentelemetry-semconv")
   implementation("io.opentelemetry.proto:opentelemetry-proto")
+  implementation("io.opentelemetry:opentelemetry-sdk-logs:1.9.0-alpha")
```

Comment:
Isn't this present in the otel alpha bom? Do you need to specify version explicitly?

Reply:
This pr was created before the agent was update to 1.9 sdk, perhaps now after rebase the version can be removed.